components/carousel/index.tsx

```diff
@@ -21,13 +21,15 @@ export interface CarouselProps extends Settings {
   slickGoTo?: number;
   dotPosition?: DotPosition;
   children?: React.ReactNode;
+  dotsClass?: string;
 }
 
 export default class Carousel extends React.Component<CarouselProps, {}> {
   static defaultProps = {
     dots: true,
     arrows: false,
     draggable: false,
+    dotsClass: '',
```

Comment:
default null?

Reply:
this is the fallback condition

Reply:
Let me know when can I expected to be merge.

Reply:
Also, let me know if any changes required from my side in this PR.

Reply:
<img width="730" alt="image" src="https://user-images.githubusercontent.com/507615/73620089-34d46080-466b-11ea-865e-2324fba968d7.png">

Reply:
all issue resolved